Automattic/wp-calypso
# fix(email-providers): add price to email management page

Related to https://github.com/Automattic/wp-calypso/issues/80967

## Proposed Changes
The issue mentioned above was previously assigned to Letero (partially) and I decided that it's good time to do Letero's part.

With this PR we display renewal price information on the Titan/GW subscription page, to have such information available on this main page of the subscription. 
More details in [the task](https://github.com/Automattic/wp-calypso/issues/80967). 

## Testing Instructions
1) Purchase 2 domains with Titan and GW subscription
2) Open Upgrades->Domains->Select Titan and then GW subscription
3) Assert that you see new descriptions with a price info (Try to turn on/off autorenew)

<table>
<tr>
<td>BEFORE
<td>AFTER
</tr>
<tr>
<td><img width="905" alt="Screenshot 2024-09-11 at 17 18 03" src="https://github.com/user-attachments/assets/26065f96-efdc-48d5-8736-49b53535c2d8">
<td><img width="921" alt="Screenshot 2024-09-11 at 17 07 37" src="https://github.com/user-attachments/assets/b0f6cf02-f4cf-475d-9ca3-ac1e0f7937b1">
</tr>
<tr>
<td><img width="909" alt="Screenshot 2024-09-11 at 17 17 00" src="https://github.com/user-attachments/assets/484e1e91-09f5-4297-ad53-54186f01acaf">
<td><img width="1195" alt="Screenshot 2024-09-11 at 17 10 01" src="https://github.com/user-attachments/assets/8cff4fa5-c982-4414-abd0-19dc15d2b9e9">
</tr>
<tr>
<td><img width="905" alt="Screenshot 2024-09-11 at 17 16 18" src="https://github.com/user-attachments/assets/9ffa90cf-48d6-44f2-8dae-c7b8eab00d52">
<td><img width="909" alt="Screenshot 2024-09-11 at 17 08 45" src="https://github.com/user-attachments/assets/8dcd95d0-7a0d-4d9d-8a5f-97b695feb2a5">
</tr>
<tr>
<td><img width="905" alt="Screenshot 2024-09-11 at 17 15 29" src="https://github.com/user-attachments/assets/eff80da3-c9e4-492f-995a-540becfda2d3">
<td><img width="910" alt="Screenshot 2024-09-11 at 17 13 15" src="https://github.com/user-attachments/assets/1fdb56fb-f781-474e-b76f-7c686d6d2d89">
</tr>
</table>
